setup.sh

```diff
@@ -158,48 +158,53 @@ website
 ###############
 # Set up auth #
 ###############
-echo ""
-read -p "Would you like to configure $(tput bold)$(tput setaf 3)end-user authentication?$(tput sgr0) (y/n) " auth_yesno
-
-if [[ ${auth_yesno} == "y" ]]; then
-    sh ./scripts/configure_auth.sh
-else
-    echo "Skipping end-user authentication configuration. You can configure it later by running:"
-    echo ""
-    echo "  export $(tput bold)PROD_PROJECT$(tput sgr0)=$(tput setaf 6)${PROD_PROJECT}$(tput sgr0)"
-    echo "  export $(tput bold)STAGE_PROJECT$(tput sgr0)=$(tput setaf 6)${STAGE_PROJECT}$(tput sgr0)"
-    echo "  export $(tput bold)OPS_PROJECT$(tput sgr0)=$(tput setaf 6)${OPS_PROJECT}$(tput sgr0)"
-    echo "  $(tput setaf 6)sh scripts/configure_auth.sh$(tput sgr0)"
+if [[ -z "${SKIP_AUTH}" ]]; then
     echo ""
+    read -p "Would you like to configure $(tput bold)$(tput setaf 3)end-user authentication?$(tput sgr0) (y/n) " auth_yesno
+
+    if [[ ${auth_yesno} == "y" ]]; then
+        sh ./scripts/configure_auth.sh
+    else
+        echo "Skipping end-user authentication configuration. You can configure it later by running:"
+        echo ""
+        echo "  export $(tput bold)PROD_PROJECT$(tput sgr0)=$(tput setaf 6)${PROD_PROJECT}$(tput sgr0)"
+        echo "  export $(tput bold)STAGE_PROJECT$(tput sgr0)=$(tput setaf 6)${STAGE_PROJECT}$(tput sgr0)"
+        echo "  export $(tput bold)OPS_PROJECT$(tput sgr0)=$(tput setaf 6)${OPS_PROJECT}$(tput sgr0)"
+        echo "  $(tput setaf 6)sh scripts/configure_auth.sh$(tput sgr0)"
+        echo ""
+    fi
 fi
 
 ################
 # Set up CI/CD #
 ################
 
-REPO_CONNECT_URL="https://console.cloud.google.com/cloud-build/triggers/connect?\
-project=${OPS_PROJECT}"
-echo "Connect your repos: ${REPO_CONNECT_URL}"
-python3 -m webbrowser ${REPO_CONNECT_URL}
+if [[ -z "${SKIP_REPO_CONNECTION}" ]]; then
```

Comment:
Per #364, we need to remove the repo connection steps from setup.sh and make sure script/terraform is updated to take care of things e2e. I assume that means needing to get the new terraform experiment into place.

I think that means: doing this is fine to unblock testing, but we shouldn't spend much time on it as it will be removed in 2-3 weeks.